source/grid/tria.cc

```diff
@@ -2092,6 +2092,12 @@ namespace internal
                                            tria_level.refine_flags.size(),
                                          /*RefinementCase::no_refinement=*/0);
 
+          tria_level.refine_choice.reserve(total_cells);
+          tria_level.refine_choice.insert(
+            tria_level.refine_choice.end(),
+            total_cells - tria_level.refine_choice.size(),
+            /*ReferenceCell::isotropic_refinement=*/0);
```

Comment:
Is this not working?
```suggestion
            ReferenceCell::isotropic_refinement);
```